test/test-rdatasets.rb

```diff
@@ -127,6 +127,50 @@ def setup
       end
     end
 
+    sub_test_case("airquality") do
+      def setup
+        @dataset = Datasets::Rdatasets.new("datasets", "airquality")
+      end
+
+      test("#each") do
+        records = @dataset.each.to_a
+        assert_equal([
+                       153,
+                       {:Ozone=>nil, :"Solar.R"=>nil, :Wind=>14.3, :Temp=>56, :Month=>5, :Day=>5},
+                       {:Ozone=>20, :"Solar.R"=>223, :Wind=>11.5, :Temp=>68, :Month=>9, :Day=>30},
+                     ],
+                     [
+                       records.size,
+                       records[4],
+                       records[-1]
+                     ])
+      end
+    end
+
+    sub_test_case('attenu') do
+      def setup
+        @dataset = Datasets::Rdatasets.new('datasets', 'attenu')
+      end
+
+      test('#each') do
+        records = @dataset.each.to_a
+        assert_equal([
+                       182,
+                       {:event=>1, :mag=>7, :station=>117, :dist=>12, :accel=>0.359},
+                       { event: 16, mag: 5.1, station: nil, dist: 7.6, accel: 0.28 },
+                       {:event=>23, :mag=>5.3, :station=>"c168", :dist=>25.3, :accel=>0.23},
+                       { event: 23, mag: 5.3, station: 5072, dist: 53.1, accel: 0.022 }
```

Comment:
Why do you mix `{:XXX => ...}` style and `{XXX: ...}` style?

Reply:
Thanks :+1:
It's cleaner code!! I will change to yours.

In the test, I didn't realize because I just copy-and-pasted, my attention went to the value only. Thanks again.